torchdynamo/config.py

```diff
@@ -95,4 +92,4 @@
 
 # Not all backends support scalars. Some calls on torch.Tensor (like .item()) return a scalar type.
 # When this flag is set to False, we introduce a graph break instead of capturing.
-capture_scalar_outputs = False
+capture_scalar_outputs = True
```

Comment:
I would make sure this is an okay default - I am not sure every backend is alright with this. If it is, we should just pull the config, as I made it config driven out of an abundance of caution w/r/t backends.

Reply:
Yeah I don't think I need to make this the default. I can leave it up to the user to decide whether they want to use this